lib/kitchen/directions/book_answer_key_container/main.rb

```diff
@@ -3,8 +3,8 @@
 module Kitchen
   module Directions
     module BookAnswerKeyContainer
-      def self.v1(book:)
-        V1.new.bake(book: book)
+      def self.v1(book:, klass: 'solutions')
+        V1.new.bake(book: book, klass: klass)
```

Comment:
In Sociology class used in the Answer Key differs (it is `solution`, not solutions), I have left `solutions` as the default value, so it can be still used by other books which have written answer key strategy without breaking the baking process